### DISCUSSION TITLE
EAGIN error when installing prisma @4.6.1 on cPanel

### DISCUSSION THREAD

**Discussion by sspahic2** [prisma/prisma#16365] [Q&A]
Hello everyone,
I usually don't post questions, but I've exhausted every google post and possible fix that I could find.

The issue is that when running **npm install** in my project on cPanel, I keep getting the same error over and over again. Here is the last part of the log message that I'm receiving.

![image](https://user-images.githubusercontent.com/12472045/202915389-4aab034b-7f9c-4184-9f88-15e0ba90c599.png)

I'm using **prisma @4.6.1** and my **node version is 16.17.1**, since that is what my hosting provider is requiring.

Also here is my **package.json**.

![image](https://user-images.githubusercontent.com/12472045/202915582-322ae259-0e3c-4163-bb22-5944bcc469a8.png)

Maybe the issue is with me using nextjs. To deploy it a server.js file needs to be created and used. That limits nextjs functionalities, but it seems to be the only way to deploy it to cPanel.

My hope, if this is not the place to ask this question, is to just narrow down where the issue is.

**Comment by nurul3101** [comment]
Hello @sspahic2 👋 

I haven't come across a similar issue before, have you configured any [binaryTargets](https://www.prisma.io/docs/reference/api-reference/prisma-schema-reference#binarytargets-options) in your schema file?

**Reply by sspahic2** [reply]
Hello @nurul3101 ,
I did not. This is the first time that I'm seeing that option. But I believe there is a more underlying issue.

Brief updated: I managed to install prisma@4.6.1. Luckily when running npx prisma generate, it prompted that I was missing the entire prisma package. So now when I run npm install, the preinstall returns code 0, which is good.

The issue now is that node is still returning either code 134 or code 1. I think these codes mean that when running npm install, some package wasn't able to be installed.

My guess this isn't a prisma issue, but npm, or neither. Hope that it's something really dumb.
I'll post my question on npm and check there.

**Reply by nurul3101** [reply]
Does this happen if you use other package managers like yarn or pnpm?

**Reply by sspahicparagon** [reply]
Oh you see, I didn't try that. Yeah something like yarn, pnpm, nvm etc. I can give that a try.

I deployed my app on Heroku, just to check if it works and yeah. Went up in 5 mins. So it's definitely something cPanel related.

Anyways, thank you for your suggestions. Don't want to waste anymore of your time. I'll try a different package manager and write the result here. Then close it.

lots of love @nurul3101 <3

**Reply by nurul3101** [reply]
No worries at all. Glad to be able to help 😃 
Sure, do let us know if a different package manager works for you.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
